gap/PolygonalComplexes/modification.gd

```diff
@@ -1206,6 +1235,55 @@ DeclareOperation( "TetrahedralExtension", [IsSimplicialSurface, IsPosInt] );
 DeclareOperation( "EdgeTurn", [IsSimplicialSurface, IsPosInt,IsPosInt] );
 #! @EndGroup
 
+#! @BeginGroup BuildingBlocks
+#! @Description
+#! Return the building blocks of a vertex-faithful simplicial sphere.
+
+#! Given two closed simplicial surfaces with euler-characteristic 2 a new surface
+#! can be constructed by computing the connected-face sum 
+#! (<Ref Subsect="ConnectedFaceSum"/>) of this surfaces.
+#! This gives rise to a waist of length 3 in the resulting surface.
+#! Applying this procedure for example to two tetradron with any two faces 
```

Comment:
- Here is a typo. 
- In my opinion it is not useful to have an example in the description. But it would be nice to write these things to the example below.